### ISSUE TITLE
Bubbly Fissure in Desolace

### ISSUE THREAD

**Issue by AgeAgainst** [dalaranwow/dalaran-wow#4123]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
When you go to a Bubbly Fissure in Desolace, it does not give the breathing buff to fill up your breath bar.  Multiple locations.
**Expected behaviour**: Tell us what should happen instead
Go to Bubbly Fissure, get breath bar filled
**Steps to reproduce the problem**:

1. Go to Bubbly Fissure in Desolace (image below shows the one at 33.2, 31.7)
2. Breath bar does not fill to full.
3. 

**Include proofs for this behaviour**
https://s24.postimg.org/bmxfbgcfp/bubbly_fissure.jpg

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
I was unable to find the ID for Bubbly Fissure on wowhead, but I did find a comment in this link (search for bubbly):
http://www.wowhead.com/quest=6142/clam-bait#comments:id=131203
http://www.wowhead.com/object=3737/bubbling-fissure This object acts the same way as Bubbly Fissure.
http://db.vanillagaming.org/?object=177524  This is from a vanilla db in which there is no change in WoTLK to the best of my knowledge in Desolace, but it marks the spots of the Bubbly Fissure. Not sure if this is helpful.

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/1010
